lnbits/core/templates/core/wallet.html

```diff
@@ -4,7 +4,7 @@
 <!---->
 {% block scripts %} {{ window_vars(user, wallet) }}{% endblock %} {% block page
 %}
-<div class="row q-col-gutter-md">
+<div class="row q-col-gutter-md" style="margin-bottom: 6rem">
```

Comment:
```suggestion 
<div class="row q-col-gutter-md q-mb-xl">
```

Reply:
it doesn't work, or I would have done it with classes!!